client/blocks/inline-help/style.scss

```diff
@@ -0,0 +1,257 @@
+	&.is-top-left .popover__arrow:before,
+	&.is-top-right .popover__arrow:before {
+		border-top: 10px solid #f3f6f8;
+	}
+
+	.button.inline-help__button {
+		text-align: left;
+		font-weight: 400;
+		width: 100%;
+		background: $gray-light;
+		border-top: 1px solid lighten( $gray, 20% );
+		padding: 16px 16px 16px 48px;
+		font-size: 16px;
+		line-height: 1;
+		position: relative;
+		border-radius: 0 0 4px 4px;
+
+		@include breakpoint( ">660px" ) {
+			font-size: 14px;
+			padding: 12px 16px 12px 48px;
+		}
+
+		.gridicon {
+			position: absolute;
+				top: 13px;
+				left: 13px;
+
+			@include breakpoint( ">660px" ) {
+				top: 8px;
+			}
+		}
+	}
+}
+
+.inline-help__heading {
+	.search-card {
+		border-radius: 4px 4px 0 0;
+		margin: 0;
+
+		.search {
+			border-radius: 4px 4px 0 0;
+			height: auto;
+
+			&.is-open.has-focus {
+				box-shadow: 0 0 0 1px $blue-wordpress,
+					0 0 0 6px $blue-light;
+
+				@include breakpoint( ">660px" ) {
+					box-shadow: 0 0 0 1px $blue-wordpress,
+						0 0 0 4px $blue-light;
+				}
+			}
+
+			.search__input {
+				font-size: 18px;
+				padding: 12px 0;
+				color: $gray-text;
+
+				@include breakpoint( ">660px" ) {
+					font-size: 15px;
+					padding: 16px 0;
+				}
+			}
+
+			.search__icon-navigation:focus {
+				border-radius: 3px;
+				box-shadow: 0 0 0 3px $blue-light;
+			}
+		}
+	}
+
+	.button {
+		vertical-align: middle;
+	}
+}
+
+.inline-help__results-list {
+	list-style: none;
+	text-align: left;
+	margin: 0;
+	min-height: 110px;
+
+	.inline-help__results-item {
+		margin: 0;
+		font-size: 18px;
+		border-top: 1px solid $border-ultra-light-gray;
+
+		@include breakpoint( ">660px" ) {
+			font-size: 14px;
+			line-height: 1.3;
+		}
+
+		button {
+			color: $blue-wordpress;
+			text-decoration: underline;
+			font-weight: normal;
+			line-height: 1.4;
+			display: block;
+			padding: 14px 16px;
+			
+			&:hover {
+				cursor: pointer;
+			}
+		}
+
+		&.is-selected {
+			background: $blue-medium;
+			cursor: pointer;
+			color: $white;
+
+			button {
+				color: $white;
+			}
+		}
+	}
+}
+
+.inline-help__empty-results {
+	padding: 16px 16px 0 16px;
+	margin: 0;
+	font-size: 14px;
+	color: $gray;
+	font-style: italic;
+
+	@include breakpoint( "<660px" ) {
+		font-size: 16px;
+	}
+}
+
+.inline-help__results-placeholder {
+	margin: 24px 16px;
+	min-height: 188px - 48px;;
+
+	.inline-help__results-placeholder-item {
+		//margin-bottom: 0.5em;
+		height: 16px;
+		margin: 16px 0;
+		border-radius: 16px;
+		background-color: lighten( $gray, 28 );
+		color: transparent;
+		animation: inline-help__results-placeholder-loading 2s cubic-bezier(0.175, 0.885, 0.32, 1.275) infinite;
+		animation-delay: 0s;
+
+		&:nth-child( 1 ) {
+			animation-delay: -1.3s;
+		}
+
+		&:nth-child( 2 ) {
+			animation-delay: -2.5s;
+		}
+
+		&:nth-child( 3 ) {
+			animation-delay: -3.8s;
+		}
+
+		&:after {
+			content: '\00a0';
+		}
+	}
+}
+
+@keyframes inline-help__results-placeholder-loading {
```

Comment:
This looks like a good candidate to be made into an easy to use component `PlaceholderLines`